contracts/bancor-portal/BancorPortal.sol

```diff
@@ -108,7 +111,8 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
         IUniswapV2Router02 uniswapV2Router,
```

Comment:
We typically name these input arguments: `initUniswapV2Router`, `initUniswapV2Factory`, etc.
This allows us to implement getter functions with the names that you are currently using, without collisions.